packages/-ember-data/tests/integration/adapter/record-persistence-test.js

```diff
@@ -42,27 +42,18 @@ module('integration/adapter/record_persistence - Persisting Records', function(h
       return run(RSVP, 'resolve');
     };
 
-    run(() => {
-      store.push({
-        data: {
-          type: 'person',
-          id: '1',
-          attributes: {
-            name: 'Braaaahm Dale',
-          },
+    const tom = store.push({
```

Comment:
Nbd but I don't think we decided to switch to `const` for normal variables

Reply:
We typically use `const` when it should not change which is the case here